datasets/_externals/ext_gb_fca_firds.yml

```diff
@@ -0,0 +1,83 @@
+type: external
+title: UK Financial Instruments Reference Data System (FIRDS)
+entry_point: zavod.runner.local_enricher:enrich
+full_dataset: gb_fca_firds
+disabled: false
+hidden: false
```

Comment:
I think we can leave these out normally and only add them when we're setting them to something other than defaut . That way most crawlers have slimmer metadata and less noise